cmd/evmosd/testnet.go

```diff
@@ -148,6 +154,18 @@ Example:
 			args.startingIPAddress, _ = cmd.Flags().GetString(flagStartingIPAddress)
 			args.numValidators, _ = cmd.Flags().GetInt(flagNumValidators)
 			args.algo, _ = cmd.Flags().GetString(flags.FlagKeyType)
+			baseFee, _ := cmd.Flags().GetString(flagBaseFee)
+			minGasPrice, _ := cmd.Flags().GetString(flagMinGasPrice)
+
+			ok := false
+			args.baseFee, ok = sdk.NewIntFromString(baseFee)
+			if !ok {
+				return fmt.Errorf("invalid value for --base-fee. expected an int; got %s", baseFee)
+			}
+			args.minGasPrice, err = sdk.NewDecFromStr(minGasPrice)
+			if err != nil {
+				return fmt.Errorf("invalid value for --min-gas-price. expected an int or decimal; got %s", minGasPrice)
+			}
```

Comment:
Please add validation that these values are > 0.
Currently, these commands work and set negative `feemarket.params.min_gas_price` or `base_fee`

```
evmosd testnet init-files --v 1 --output-dir ./nodes --chain-id evmos_9000-1 --keyring-backend test --base-fee 10000 --min-gas-price -80000
evmosd testnet init-files --v 1 --output-dir ./nodes --chain-id evmos_9000-1 --keyring-backend test --base-fee -10000 --min-gas-price 80000
```

Reply:
A non-negative check has been made. But the Uint64 type cannot be used, because baseFee can be a large array, such as 10000000000000gwei, and Uint64 cannot meet its needs at this time.